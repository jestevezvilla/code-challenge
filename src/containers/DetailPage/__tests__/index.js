import React from 'react';
import { shallow } from 'enzyme';
import { DetailPage } from '../index';

it('renders without crashing', () => {
  shallow(<DetailPage />);
});
