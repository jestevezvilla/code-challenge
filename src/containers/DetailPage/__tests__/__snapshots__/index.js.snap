// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailPage should snapshot be OK 1`] = `
<div
  className="DetailPage"
>
  <Loader
    title="Loading data..."
  />
</div>
`;
